values.yaml

```diff
@@ -255,8 +251,26 @@ webserver:
     lastName: user
     password: admin
 
+  # Mount additional volumes into webserver.
+  extraVolumes: []
+  extraVolumeMounts: []
+
+  # This will be mounted into the Airflow Webserver as
+  # a custom webserver_config.py
+  webserverConfig: ~
```

Comment:
Hmmm, I wonder if it is possible to get the feature we want without having this as a field in the helm chart -- it's a bit odd to have a "large" block like this, and most people would be able to bake the webserver_config.py they need in to their images.

Reply:
Yea I agree its a bit awkward. I guess it could technically be supported using the `volume` mount options right above it, assuming they create the file in their own `ConfigMap` or `Secret`.